src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -398,12 +398,10 @@ private static String replaceUnpredictableValuesWithPlaceholders(String content)
                       .replace(TimeHelper.formatDate(now), "${today}")
                       // date/time now e.g [Thu, 07 May 2015, 07:52 PM] or [Thu, 07 May 2015, 07:52 PM UTC]
                       .replaceAll(dateTimeNow + REGEX_DISPLAY_TIME, "\\${datetime\\.now}")
-                      // jQuery files
-                      .replace("/js/lib/jquery.min.js", "${lib.path}/jquery.min.js")
-                      .replace("https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js", "${lib.path}/jquery.min.js")
-                      // jQuery-ui files
-                      .replace("/js/lib/jquery-ui.min.js", "${lib.path}/jquery-ui.min.js")
-                      .replace("https://ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/jquery-ui.min.js", "${lib.path}/jquery-ui.min.js")
+                      // jQuery js file
+                      .replace(Const.SystemParams.getjQueryFilePath(TP.isDevServer()), "${lib.path}/jquery.min.js")
```

Comment:
Nope. `SystemProperty.*` (`*` can e.g be `applicationId`, `applicationVersion`, or `environment`) are meaningful only when called by a GAE-compiled class inside the application (I don't know how to better describe it, but it would include most of our production codes). Calls from any other random Java class (and this includes all our tests) will simply result in null.
You can try it yourself: try to display `SystemProperty.*.get()` from any of the test classes. I can guarantee you a null. Better yet, try and call  `Config.inst().getApplicationVersion()`: you'll get an NPE. (yes this is another bug that I've known for ages, but it's not so poisonous so I still let it)